### PR TITLE
addresses active reported issues as reported by CodeQL

### PR DESCRIPTION
**Pull request by tlgevers**
Issues addressed within this PR: 
[222](https://github.com/protectai/modelscan/issues/222) Unused import - file: tools/picklescanner.py
[223](https://github.com/protectai/modelscan/issues/223) Unused import - file: skip.py
[224](https://github.com/protectai/modelscan/issues/224) Missing call to __init__ - file: issues.py
[225](https://github.com/protectai/modelscan/issues/225) Potentially uninitialized local variable - file: cli.py
[226](https://github.com/protectai/modelscan/issues/226) File is not always closed- file: cli.py
[227](https://github.com/protectai/modelscan/issues/227) File is not always closed - file: model.py

### CODE REVIEW

File: modelscan/cli.py [hunk 1]
```diff
@@ -115,6 +115,7 @@ def scan(
         with open(settings_file_path, encoding="utf-8") as sf:
             settings = parse(sf.read()).unwrap()
             click.echo(f"Detected settings file. Using {settings_file_path}. \n")
+            sf.close()
```
Comment (iamfaisalkhan) [inline modelscan/cli.py:118]: The `close` should be redundant as the context manager should close it.

**Review by iamfaisalkhan — COMMENTED** [review]
Most of the changes looks good. Explicit close within `context` appear redundant. Also do you mind, running `make format` to fix some of the issues related to formatting.

File: modelscan/cli.py (hunk 1 above)
Comment (tlgevers) [reply modelscan/cli.py]: Thank you Faisal, I have updated the PR, removed the redundant close & also ran make format to ensure formatting is corrected.

**Review by iamfaisalkhan — APPROVED** [review]
LGTM 🚀

**Pull request merged**